### PR TITLE
initial commit of guide for producing a release blog post and video

### PR DESCRIPTION
**Pull request by shannonbradshaw**
Do not merge.

Updates will follow.

Just filing this PR so we can begin to discuss.

### CODE REVIEW

**Comment by jbottum** [comment]
@kimwnasptd @annajung I believe this PR documents an initial version of our process.     The are some possible small nits but I think it is functional.    Would you please review and comment or /lgtm ?   I would like to wrap this up by next Tuesday's Community call if possible.   Thanks.  Josh

File: releases/release-announcement-guide.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+# Kubeflow’s Release Process Addendum - Release Blog Post and Presentation
+
+
+# Purpose
+
+This document reviews the Kubeflow Community’s process for a blog post and a recorded presentation, which are delivered with each Kubeflow release.   The blog post and presentation are used to increase the Community’s understanding of new features in a Kubeflow release.
```
Comment (annajung) [inline releases/release-announcement-guide.md:6]:
```suggestion
This document reviews the Kubeflow Community’s process for a blog post and a recorded presentation delivered with each Kubeflow release. The blog post and presentation are used to increase the Community’s understanding of new features in a Kubeflow release.  
```

File: releases/release-announcement-guide.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+
+* Product Manager - 20 hours
+    * Manages the blog’s creation and release process,  creates initial version, asks and incorporates edits, keeps process moving, provides updates to Community
+* GitHub specialist - 10 hours
+    * Creates PR, helps to check content, incorporate edits to PR
+* Content creators, editors and approvers - 2 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Final Approver - 4 hours
+    * Kubeflow Steering Committee lead
```
Comment (annajung) [inline releases/release-announcement-guide.md:38]:
```suggestion
    * [Kubeflow Steering Committee](https://github.com/kubeflow/internal-acls/blob/9f5423223bd24aaa236877415014d77aa8aa1fd6/github-orgs/kubeflow/org.yaml#L917)
```

File: releases/release-announcement-guide.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+An example of the slide is available here: [https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651](https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651)
+
+
+# Sample Activity Timeline
+
+This timeline shows potential activities for the blog and presentation, starting from 35 days from the postings.
+
+
+<table>
```
Comment (annajung) [inline releases/release-announcement-guide.md:82]: I found this table really hard to follow especially with the time being part of the content. Would it make sense to have a separate column for time? 

Also, while I can follow the requirements, I am not sure who is completing these actions listed. Could we add who is responsible for the tasks? Something similar to below?

| Time | Who | Blog | Presentation |
| -- | -- | --| -- |
| 35 days before the release | Product Manager | action for blog post | action for presentation |

File: releases/release-announcement-guide.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+    * We need a pro at editing videos (that are currently created via Zoom)
+* Final Approver - 2 hours
+    * Kubeflow Steering Committee lead
+
+
+## Example: 
+
+An example of the recorded presentation is here: [https://www.youtube.com/watch?v=gG61gHw4J14](https://www.youtube.com/watch?v=gG61gHw4J14)
+
+An example of the slide is available here: [https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651](https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651)
```
Comment (annajung) [inline releases/release-announcement-guide.md:74]:
```suggestion
- [Kubeflow 1.4 Release Presentation](https://www.youtube.com/watch?v=gG61gHw4J14)
- [Kubeflow 1.4 Release Presentation Slides](https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651)
```

File: releases/release-announcement-guide.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+  <tr>
+   <td>3 days out
+<p>
+Get final /ltgm from stakeholders
+<p>
+Steering Committee adds Final edits
+   </td>
+   <td>3 days out 
+<p>
+Submit to Steering committee for posting on Kubeflow channel on YouTube
```
Comment (annajung) [inline releases/release-announcement-guide.md:287]: Can you add how to submit the video to the steering committee?

File: releases/release-announcement-guide.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+<p>
+Ask all Resources to review for any corrections
+<p>
+Have team ready for an quick updates
+<p>
+Promote on Kubeflow slack and on Kubeflow-discuss
+   </td>
+   <td>Day of Post
+<p>
+Highlight in presentation on Kubeflow-discuss mailing list (<a href="mailto:kubeflow-discuss@googlegroups.com">kubeflow-discuss@googlegroups.com</a> and on Kubeflow-slack (kubeflow.slack.com) in general channel
```
Comment (annajung) [inline releases/release-announcement-guide.md:303]: - Could we also highlight the release in [kubeflow twitter](https://twitter.com/kubeflow)?
- Are we able to leverage the slack `#announcement` channel?

File: releases/release-announcement-guide.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>11 days out
+<p>
+Review Working Group edits
+<p>
+Create and distribute 2rd version to Working Groups
```
Comment (annajung) [inline releases/release-announcement-guide.md:197]:
```suggestion
Create and distribute 2nd version to Working Groups
```
I don't think I follow here, 2nd version of what?

File: releases/release-announcement-guide.md [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the blog’s creation and release process,  creates initial version, asks and incorporates edits, keeps process moving, provides updates to Community
+* GitHub specialist - 10 hours
+    * Creates PR, helps to check content, incorporate edits to PR
```
Comment (annajung) [inline releases/release-announcement-guide.md:34]: Could you clarify why a GitHub specialist is needed? Couldn't we combine these two roles since the product manager is the one who writes and reiterates the post based on feedback?

If the concern is that the product manager may not have git skills, I think we should also consider the possibility that GitHub specialist might not be able to edit the PR as effectively as someone who understand the content of the feedback

File: releases/release-announcement-guide.md [hunk 9]
```diff
@@ -0,0 +1,306 @@
...
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the process, creates initial presentation outline, asks working groups for content and speakers, reviews and incorporates edits, keeps, provides updates to Community, keeps the process moving
+* Content creators, editors and approvers - 3 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Video Editor
+    * We need a pro at editing videos (that are currently created via Zoom)
```
Comment (annajung) [inline releases/release-announcement-guide.md:65]: I think it might be sufficient to say someone with basic editing skills.

File: releases/release-announcement-guide.md [hunk 10]
```diff
@@ -0,0 +1,306 @@
...
+## Example: 
+
+An example of the recorded presentation is here: [https://www.youtube.com/watch?v=gG61gHw4J14](https://www.youtube.com/watch?v=gG61gHw4J14)
+
+An example of the slide is available here: [https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651](https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651)
+
+
+# Sample Activity Timeline
+
+This timeline shows potential activities for the blog and presentation, starting from 35 days from the postings.
```
Comment (annajung) [inline releases/release-announcement-guide.md:79]: Clarification of posting date would help someone reading this guideline to plan

**Review by annajung — CHANGES_REQUESTED** [review]
I left some nit comments which not need to be addressed for the initial version. However, I think the table needs to be reiterated on. I found the table really hard to follow and I think that's the key component of this documentation.

File: releases/release-announcement-guide.md (hunk 8 above)
Comment (jbottum) [reply releases/release-announcement-guide.md:34]: I documented the current process.   If someone has the git and kubeflow content skills then then the GitHub specialists role could be collapsed into the PM.   Note - It helps to have two people working together because last minute changes and updates often cause re-writes.

File: releases/release-announcement-guide.md (hunk 2 above)
Comment (jbottum) [reply releases/release-announcement-guide.md:38]: ok

File: releases/release-announcement-guide.md (hunk 9 above)
Comment (jbottum) [reply releases/release-announcement-guide.md:65]: ok

File: releases/release-announcement-guide.md (hunk 4 above)
Comment (jbottum) [reply releases/release-announcement-guide.md:74]: ok

File: releases/release-announcement-guide.md (hunk 1 above)
Comment (jbottum) [reply releases/release-announcement-guide.md:6]: /lgtm

**Comment by jbottum** [comment]
@shannonbradshaw I hit "start a review" by mistake.   I think that we should continue to modify based on Anna's comments.    Want would be the best way forward...create a new PR to modify or un-commit this...I think we might un-commit because we have Anna's comments here.
